test/e2e/ensure_pod_disruption_budgets_exist_test.go

```diff
@@ -0,0 +1,78 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kube-reporting/metering-operator/test/reportingframework"
+)
+
+func testEnsurePodDisruptionBudgetsExist(t *testing.T, rf *reportingframework.ReportingFramework) {
+	tt := []struct {
+		Name         string
+		Exists       bool
+		MinAvailable int32
+	}{
+		{
+			Name:         "presto-coordinator-pdb",
+			Exists:       true,
+			MinAvailable: 1,
+		},
+		{
+			Name:         "presto-worker-pdb",
+			MinAvailable: 0,
+		},
+		{
+			Name:         "reporting-operator-pdb",
+			Exists:       true,
+			MinAvailable: 1,
+		},
+		{
+			Name:         "hive-metastore-pdb",
+			Exists:       true,
+			MinAvailable: 1,
+		},
+		{
+			Name:         "hive-server-pdb",
+			Exists:       true,
+			MinAvailable: 1,
+		},
```

Comment:
We probably want to do something a bit more sophisticated than this, like read the MeteringConfig for the correct values, but this is sufficient for now.

Reply:
Can an e2e test drain a node and then check for continued functionality of the operand that was on that node?

Reply:
That should be possible in theory as we're using a kubeadmin kube client for the testing suite. Something we may want to do instead (or supplemental to your suggestion) is to patch the MeteringConfig custom resource, disabling the PDBs for all of the operands, and ensuring those resources are properly deleted by the operator.